website/blog/2022-04-25-jest-28.md

```diff
@@ -53,7 +53,7 @@ Jest now ships with a reporter to be used on GitHub Actions, which will use anno
 
 ![GitHub Actions test error screenshot](/img/blog/28-gh-actions-reporter.png)
 
-You can activate this reporter by passing `github-actions` in the [`reporters` configuration option](docs/configuration#reporters-arraymodulename--modulename-options).
+You can activate this reporter by passing `github-actions` in the [`reporters` configuration option](/docs/configuration#reporters-arraymodulename--modulename-options).
```

Comment:
btw, would it be possible to use `Configuration.md` somehow here? then docusaurus would've told me the link was broken 😅

Reply:
No, you can't link pages between plugins, as of now. We'll probably change that in v3, but no time for architectural changes in v2 now :( (https://github.com/facebook/docusaurus/issues/6370)

Reply:
cool, thanks for the context!